src/documents/ComponentLifecycle.md

```diff
@@ -61,6 +62,7 @@ Note that since JavaScript is a garbage-collected language, the component object
 although it will as noted above be detached from its parent (via a call to `delete`) and similarly all subcomponent references will be recursively detached from their parents. 
```

Comment:
This is another "Note" on the page and should also get the "infusion-docs-note" styling.